README.md

```diff
@@ -45,7 +45,8 @@ Each release of this client will support the equivalent Dgraph release. For exam
 Make a new client by passing in one or more GRPC channels pointing to alphas.
 
 ```c#
-var client = new DgraphClient(new Channel("127.0.0.1:9080", ChannelCredentials.Insecure));
+AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
+var client = new DgraphClient(GrpcChannel.ForAddress("http://localhost:9080"), new GrpcChannelOptions { Credentials = ChannelCredentials.Insecure }));
```

Comment:
DgraphClient client only accepts one parameter. Did you mean to put the GrpcChannelOptions inside of the ForAddress method?